src/plugins/intel_cpu/tests/functional/test_utils/filter_cpu_info.hpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <string>
+
+namespace CPUTestUtils {
```

Comment:
Why do we need to move this to a new header file?

Reply:
fixed